.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1103,8 +1151,11 @@ stages:
         dockerComposeFileArgs: |
           baseImage=$(baseImage)
           framework=$(publishTargetFramework)
+          DD_LOGGER_DD_API_KEY=$(DD_LOGGER_DD_API_KEY)
         dockerComposeCommand: run --no-deps --rm -e baseImage=$(baseImage) -e framework=$(publishTargetFramework) -e IncludeTestsRequiringDocker=false IntegrationTests
         projectName: ddtrace_$(Build.BuildNumber)
+      env:
+        DD_LOGGER_DD_API_KEY: $(ddApiKey)
```

Comment:
I'm surprised this works but I'll take your word for it :D I thought you would have to pass it using `-e` like for `baseImage` and `framework` above 🤔

Reply:
yes, it works (otherwise we would have a crash due to missing API Key). Now I wonder if we actually need the `-e` parameters

Reply:
> otherwise we would have a crash due to missing API Key

Oh, we don't want that 🙁 we need to be able to run this locally _without_ having to pass a bunch of extra env vars... 

What crashes, exactly? If it's the logger, can we disable the logger by default in docker-compose.yml, and set an extra variable here to enable it?

Reply:
oh, that's true, I'll change that to a warning/info message in stdout in the logger repo to avoid the crash and just bypass. (introducing a bypass API key env-var in the logger).